tests/unit/lms/services/lti_grading/_v11_test.py

```diff
@@ -9,88 +9,162 @@
 from lms.services.lti_grading._v11 import LTI11GradingService
 from tests import factories
 
-pytestmark = pytest.mark.usefixtures("oauth1_service", "http_service")
 
+class GradingResponse(dict):
+    """An LTI grading response dict with convenience accessors."""
 
-class TestLTI11GradingService:
-    SERVICE_URL = "http://example.com/service_url"
-    GRADING_ID = "lis_result_sourcedid"
-
-    def test_read_result_sends_expected_request(self, svc, respond_with, http_service):
-        respond_with(score=0.95)
-
-        svc.read_result(self.GRADING_ID)
-
-        self.assert_sent_header_ok(http_service)
+    def __init__(self, status_code="success", score=0.92):
+        super().__init__(
+            {
+                "imsx_POXEnvelopeResponse": {
+                    "@xmlns": "http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0",
+                    "imsx_POXHeader": {
+                        "imsx_POXResponseHeaderInfo": {
+                            "imsx_version": "V1.0",
+                            "imsx_messageIdentifier": 1313355158804,
+                            "imsx_statusInfo": {
+                                "imsx_codeMajor": status_code,
+                                "imsx_severity": "status",
+                                "imsx_messageRefIdentifier": "999999123",
+                                "imsx_operationRefIdentifier": "readResult",
+                                "imsx_description": "An error occurred.",
+                            },
+                        }
+                    },
+                    "imsx_POXBody": {
+                        "readResultResponse": {
+                            "result": {
+                                "resultScore": {"language": "en", "textString": score}
+                            }
+                        }
+                    },
+                }
+            }
```

Comment:
I think this much more clearly shows the data structure at play than before. The strategy is we will create the full thing, and then knock bits out or modify them as required for the tests.

I've left in non-conditional things like score and status code as it's easier to do this way.